bin/bout-boutdata-restart-create

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# PYTHON_ARGCOMPLETE_OK
+
+import argparse
+try:
+    import argcomplete
+except ImportError:
+    argcomplete=None
+
+parser = argparse.ArgumentParser("Create restart files from data (dmp) files.")
+parser.add_argument("path", type=str, default=...
```

Comment:
The Ellipsis object, `...`, is for slicing. You should use `None` instead. Although these should really just use the default values from `restart.create`.

Reply:
I found the defaults, which are stored in `create.__defaults__` - but they don't contain the name of the matching argument. Which way can I get them? I don't want to that manually ...

Reply:
Found it, will fix (link to remind myself) https://stackoverflow.com/questions/2677185/how-can-i-read-a-functions-signature-including-default-argument-values#2677263

Reply:
Resolved, I have found a way to extract the default value, and am now providing them.